CONTRIBUTION.rst

```diff
@@ -0,0 +1,102 @@
+======================
+Contribution Guideline
+======================
+
+
+Welcome to the contribution guideline page!
+
+Setting up the environment
+==========================
+
+-----------------
+Installing pip
+-----------------
+This project uses `pip <https://pip.pypa.io/en/stable/>`_ as a package manager. To install, simply run the follow command:
+
+.. code-block:: bash
+
+    $ python -m pip install
+
+------------------------
+Installing pre-commit
+------------------------
+`pre-commit <https://pre-commit.com/>`_ is a multi-language package manager for pre-commit hooks. It is used to check code before it makes it to commit. A list of hooks is specified in `.pre-commit-config.yaml` in the root directory of the project.
+To install `pre-commit`, simply run the following command:
+
+.. code-block:: bash
+
+    $ pip install pre-commit
```

Comment:
Please move current:
https://github.com/RedHatQE/pylero#before-you-commit

into the contribution doc